core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/AggregatedVizOpExec.scala

```diff
@@ -0,0 +1,8 @@
+package edu.uci.ics.texera.workflow.operators.visualization
+
+import edu.uci.ics.amber.engine.common.IOperatorExecutor
+
+/**
+  * The final layer executor of an aggregated viz operator.
+  */
+trait AggregatedVizOpExec extends IOperatorExecutor {}
```

Comment:
extends from MapOpExec?

Reply:
I'm not extending from `MapOpExec` because this layer of operator might not all be Map operators?

Reply:
Then this inheritance does not make much sense. In `BarChartOpExec`, you extend both classes while `MapOpExec` is already extending `IOperatorExecutor`. My suggestion here is to remove the extends.

Reply:
It's extending `IOperatorExecutor` because it is a required type in `new WorkerLayer()`. Any better ideas for this? Or maybe just remove this trait completely and simply use `IOperatorExecutor` in `AggregatedVizOpExecConfig`?

Reply:
I think removing this trait is better since we are not actually using it anywhere.

Reply:
Done.